ContentProtection/OfflineFairPlay/Program.cs

```diff
@@ -378,7 +390,7 @@ private static async Task<Asset> CreateOutputAssetAsync(IAzureMediaServicesClien
             // This example shows how to encode from any HTTPs source URL - a new feature of the v3 API.  
             // Change the URL to any accessible HTTPs URL or SAS URL from Azure.
             JobInputHttp jobInput =
-                new JobInputHttp(files: new[] { "https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/2b533311-b215-4409-80af-529c3e853622/Ignite-short.mp4" });
+                new(files: new[] { "https://nimbuscdn-nimbuspm.streaming.mediaservices.windows.net/2b533311-b215-4409-80af-529c3e853622/Ignite-short.mp4" });
```

Comment:
Can you move this URL to a readonly constant higher up in the sample so we know where it is quickly if needed to be replaced.

Reply:
Sure. Done.